packages/file_selector/file_selector/README.md

```diff
@@ -0,0 +1,3 @@
+# file_selector
+
+A Flutter plugin that manages files and interactions with file dialogs.
```

Comment:
This README needs a huge amount of love :)

Reply:
PTAL